kuksa_certificates/jwt/createToken.py

```diff
@@ -19,10 +30,7 @@ def createJWTToken(input_filename, priv_key):
 
     encoded = jwt.encode(payload, priv_key, algorithm="RS256")
 
-    if input_filename.endswith(".json"):
-        output_filename = input_filename[:-5] + ".token"
-    else:
-        output_filename = output_filename + ".token"
+    output_filename = input_filename.removesuffix(".json") + ".token"
```

Comment:
This is both shorter and arguably more readable that the first fix.

However, `removesuffix()` was only introduced in python 3.9.0 which breaks the script on for example Ubuntu 20.04 LTS which is still used quite widely.

So I would prefer the previous version that works "everywhere".

Reply:
Agreed. reverted to "oldschool" version